### DISCUSSION TITLE
AdBlockers Block mercadopago.js

### DISCUSSION THREAD

**Discussion by edarioq** [mercadopago/sdk-react#112] [General]
From testing the sdk-react package on my local machine, I've discovered that adblockers block mercadopago.js from loading. The user is effectively unable to checkout. 

The adblocker gets triggered because it's flagged by the "sdk.mercadopago.com" domain.

Will this be an issue in production ? Has this happened to anyone else ?

**Comment by meliguicarvalho** [comment]
Hi Edgar. Thanks for bringing this to our attention.

We have no record of this happening, but will check this scenario and give you a feedback as soon as we have more information.

Can you please share more details of your implementation and which Ad Blocker you are using?

**Comment by edarioq** [comment]
I'm using Nextjs 14, using `@mercadopago/sdk-react` and just calling `initMercadoPago();` and passing it a valid public key.

I am using the latest Firefox with the Privacy Badger extension.

**Comment by meliguicarvalho** [comment]
Edgar, we are internally analyzing this situation to better understand how to avoid these issues in the future.

To answer your questions:
1 - This will happen in production for users who use this specific blocker, in this moment we cannot remedy it, as the blocking is done automatically. I assure you we are looking into how to fix it. As for right now, the user base for this is small, so this should not impact you in a significant manner. 
2 - We do not know of more instances of this happening.
